packages/spector/src/cli/cli.ts

```diff
@@ -287,20 +287,31 @@ async function main() {
             array: true,
             demandOption: true,
           })
+          .option("modes", {
```

Comment:
So I think for the manifest we don't want mode, we want either:
- each spec package upload their own maniferst and the UI does the aggregration
- you define a scenario "ruleset" which can include 1+ scenario package. In this case we'd have one standard and one azure

Reply:
The first one might be more flexible

Reply:
@timotheeguerin The first one will add logic to dashboard to aggregate i.e. to know a particular scenario is standrd/azure. I do not want to put that logic to dashboard. Let it just display the scenarios instead of classifying them. 

As for the second option i.e. creating scenario ruleset, the current code does that. If you have 2 scenario paths, one for standard and one for azure it is accomodated. So a command line such as:

`pnpm .\cmd\cli.mjs upload-manifest ..\http-specs\specs ..\..\..\typespec-azure\packages\azure-http-specs\specs\ --modes=standard --modes=azure --storageAccountName=typespec` 

will work fine. On the other hand, if some emitters want to combine more than 2 paths (for some reason)  this design will accomodate that also. To handle such cases, it would be nice to have modes here.

Reply:
thats what I mean you on don't upload 2 different modes at once only one

Reply:
either the https-specs upload its own and azure-http-specs upload its own(case 1)
or typespec repo upload its manifest(just https-specs)
and typespec-azure repo upload its manifest(aggregrated manfiest with both standard and azure) Do not upload 2 files here

Reply:
ok I see the screenshot now, yeah mode is still ireelevant it should just use the package name that produced the manifest section instgead of counting on who upoad the manifest to remap

Reply:
@timotheeguerin As discussed, added the `emitterName` instead of `modes`.